content/pages/editMetadata.json

```diff
@@ -66,6 +66,13 @@
         "type": "tags",
         "placeholder": "e.g. logistics",
         "required": false
+      },
+      {
+        "name": "paymentCollector",
+        "label": "Payment Collector Address",
+        "placeholder": "e.g. 0X123ABC...",
+        "help": "This address will receive the revenue from all sales. More info available in our [docs](TODO: Update docs + Add link here).",
```

Comment:
Docs update is in review: https://github.com/oceanprotocol/docs/pull/1153

Reply:
merged, you can update